senza/cli.py

```diff
@@ -292,119 +263,90 @@ def __init__(self, **kwargs):
 
 
 class AccountArguments:
-    '''
-    >>> test = AccountArguments('blubber',
-    ... AccountID='123456',
-    ... AccountAlias='testdummy',
-    ... Domain='test.example.org.',
-    ... TeamID='superteam')
-    >>> test.AccountID
-    '123456'
-    >>> test.AccountAlias
-    'testdummy'
-    >>> test.TeamID
```

Comment:
If to remove from doc test I would recommend to move them to proper testing code.